base/lnls-single-uid.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+#
+# Configure container filesystem so that single UID restriction is satisfied,
+# making ignore_chown_errors=True storage configuration work properly.
```

Comment:
IMO this is the sort of information that one can get directly from git history; it's clearly being done for a reason and someone who wishes to remove/understand it can git blame.

I don't know if I'm going too far on the frugal scale for docs, though, so if you'd rather keep we can leave it in.

Either way, `ignore_chown_errors=True` is not the correct configuration option. It's "systems without subuid and subgid configured"; `ignore_chown_errors=True` is part of what makes them work, but not what defines them.